README.md

```diff
@@ -34,146 +37,116 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
 Here's the quickest way to get going with Rook.
 
```

Comment:
There are two long sections that follow: Kubernetes and Standalone. Consider adding links to them here.